src/main/java/com/questhelper/helpers/quests/legendsquest/LegendsQuest.java

```diff
@@ -975,10 +975,10 @@ private void setupSteps()
 		drinkBraveryPotionAndClimbDown.addDialogSteps("Yes, I'll bravely drink the Bravery Potion.", "Yes, I'll shimmy down the rope into possible doom.", "I'm not scared. Let's go!");
 		// MesBox: You bravely swig down the entire contents of the vial and then wait for some sort of internal explosion. After a few seconds, you realise that you actually feel alright.
 
-
 		climbDownWinch = new ObjectStep(this, ObjectID.WINCH_2934, new WorldPoint(2761, 9331, 0), "Climb down the winch.");
 		climbDownWinch.addAlternateObjects(ObjectID.WINCH_2935);
-		climbDownWinch.addDialogSteps("I'm not scared. Let's go!", "Yes, I'll shimmy down the rope into possible doom.");
+		climbDownWinch.addDialogSteps("I'm not scared. Let's go!", "Yes, I'll shimmy down the rope into possible doom.", "I'm not scared. Let's go!");
```

Comment:
Undoes the duplicate "I'm not scared" dialog step
```suggestion
		climbDownWinch.addDialogSteps("I'm not scared. Let's go!", "Yes, I'll shimmy down the rope into possible doom.");
```